[2022-08-24 11:43:24,535] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.carga_arquivo manual__2022-08-24T11:43:23.183699+00:00 [queued]>
[2022-08-24 11:43:24,547] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.carga_arquivo manual__2022-08-24T11:43:23.183699+00:00 [queued]>
[2022-08-24 11:43:24,548] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-24 11:43:24,550] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-24 11:43:24,550] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-24 11:43:24,569] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): carga_arquivo> on 2022-08-24 11:43:23.183699+00:00
[2022-08-24 11:43:24,575] {standard_task_runner.py:52} INFO - Started process 74 to run task
[2022-08-24 11:43:24,579] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'raizen_lab_01', 'carga_arquivo', 'manual__2022-08-24T11:43:23.183699+00:00', '--job-id', '9713', '--raw', '--subdir', 'DAGS_FOLDER/raizen_lab_01.py', '--cfg-path', '/tmp/tmpebxh3hkz', '--error-file', '/tmp/tmp_lkpvg_x']
[2022-08-24 11:43:24,581] {standard_task_runner.py:80} INFO - Job 9713: Subtask carga_arquivo
[2022-08-24 11:43:24,667] {task_command.py:371} INFO - Running <TaskInstance: raizen_lab_01.carga_arquivo manual__2022-08-24T11:43:23.183699+00:00 [running]> on host 3f878792eb1e
[2022-08-24 11:43:24,752] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=raizen_lab_01
AIRFLOW_CTX_TASK_ID=carga_arquivo
AIRFLOW_CTX_EXECUTION_DATE=2022-08-24T11:43:23.183699+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-24T11:43:23.183699+00:00
[2022-08-24 11:43:26,309] {python.py:173} INFO - Done. Returned value was: None
[2022-08-24 11:43:26,336] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=raizen_lab_01, task_id=carga_arquivo, execution_date=20220824T114323, start_date=20220824T114324, end_date=20220824T114326
[2022-08-24 11:43:26,400] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-24 11:43:26,446] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
